### PR TITLE
data/blog: add on the node blog post

### PR DESCRIPTION
**Pull request by clairew**
On the Node While On the Road with Tailscale blog post

### CODE REVIEW

File: data/blog/on-the-node-while-on-the-road.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+One popular Tailscale feature is using another machine in your tailnet as an [exit node](https://tailscale.com/kb/1103/exit-nodes/)—an easy way to forward all internet traffic through another device. Luckily I had a Linux box at home on my tailnet that was already configured to be an exit node. I tried accessing Github using ssh with the Linux box, and it connected with no problem. This means what caused this ssh issue is related to not Github or how I configured my ssh keys, but the wifi network I was using.
+
+Most VPNs let you route traffic through another computer, but not always one that you control (and that has your known_hosts file saved). What makes using Tailscale even more compelling in this scenario is, to improve bandwidth, I can configure my exit node to only advertise routes to github.com–what we call “subnet routers”.
+
+For example, here's how you can advertise the routes for GitHub's git ingress servers:
+
+```
+$ tailscale set –-advertise-routes=”$(curl -fsSL https://api.github.com/meta | jq -r ‘.git[]’ | tr ‘\n’ ‘,’)
```
Comment (thisisparker) [inline data/blog/on-the-node-while-on-the-road.mdx:29]:
```suggestion
$ tailscale set –-advertise-routes="$(curl -fsSL https://api.github.com/meta | jq -r '.git[]' | tr '\n' ',')"
```

I think Google docs "smartened" the quotes here, right?

File: data/blog/on-the-node-while-on-the-road.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  width={872}
+  height={443}
+  alt={`Someone is doing something nasty!`}
+/>
+
+To be honest, I have never encountered this before. A fingerprint is a way for the machine to recognize the public key of a host from a previously established ssh connection. When you make connections to ssh servers, the fingerprints of every server you connect to is in your known_hosts file in ~/.ssh. The key in the known_hosts file didn't match the key that "Github" gave me. Github had [recently changed their ssh RSA host key](https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/) for security reasons, so maybe this was an opportunity of attack.
+
+One popular Tailscale feature is using another machine in your tailnet as an [exit node](https://tailscale.com/kb/1103/exit-nodes/)—an easy way to forward all internet traffic through another device. Luckily I had a Linux box at home on my tailnet that was already configured to be an exit node. I tried accessing Github using ssh with the Linux box, and it connected with no problem. This means what caused this ssh issue is related to not Github or how I configured my ssh keys, but the wifi network I was using.
```
Comment (thisisparker) [inline data/blog/on-the-node-while-on-the-road.mdx:22]:
```suggestion
I needed to figure out whether the issue was with Github's key or with my connection. To do so, I turned to a popular Tailscale feature: [exit nodes](https://tailscale.com/kb/1103/exit-nodes/), which allow you to forward all your internet traffic through another machine in your tailnet. Luckily I had a Linux box at home on my tailnet that was already configured to be an exit node. I tried accessing Github using ssh with the Linux box, and it connected with no problem. This means that what caused this ssh issue is related not to Github or how I configured my ssh keys, but to the wifi network I was using.
```

Sorry I know we'd gone back and forth on this paragraph but seeing it here gave me a thought on how to make it clearer

File: data/blog/on-the-node-while-on-the-road.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+<Figure
+  src="/images/on-the-node-while-on-the-road/exit-node-diagram.png"
+  width={880}
+  height={375}
+  alt={`Blackpink in your area.`}
+/>
+
+Here’s a diagram showing how I used my exit node. Blackpink is the linux box I used as an exit node, and keep in mind when using exit nodes if latency matters in your use case, you might want to use a machine that’s closer to you.
```
Comment (thisisparker) [inline data/blog/on-the-node-while-on-the-road.mdx:39]:
```suggestion
Here’s a diagram showing how I used my exit node. Blackpink is the linux box I used as an exit node. Keep in mind when using exit nodes if latency matters in your use case, you might want to use a machine that’s closer to you.
```

**Review by thisisparker — COMMENTED** [review]
Thank you for getting this together as a PR! The Docs -> markdown conversion can be annoying, so it's doubly appreciated. A few small changes here but then I think this is just about good to go :)

**Review by Xe — APPROVED** [review]

**Pull request merged**